src/config/fuzz_medium.mlh

```diff
@@ -0,0 +1,23 @@
+[%%import "config/ledger_depth/tiny.mlh"]
+[%%import "config/curve/medium.mlh"]
+[%%import "config/coinbase/standard.mlh"]
+(* [%%import "config/consensus/postake_full.mlh"] *)
```

Comment:
Can this line be deleted? If we should leave it, can you put a TODO so it's clear why it's here?